toonz/sources/toonz/mainwindow.cpp

```diff
@@ -2450,6 +2455,15 @@ void MainWindow::clearCacheFolder() {
 
 //-----------------------------------------------------------------------------
 
+void MainWindow::onNewAssistantLevelButtonPressed() {
```

Comment:
You're going to need to pull the latest OT changes and update this function due to the Preference changes that were recently merged since `setDefLevelType()` function no longer exists. It won't show up as a conflict but it will fail to build in Travis and Appveyor when you submit your next commit without updating it.